ovos_cli_client/text_client.py

```diff
@@ -1518,6 +1519,8 @@ def simple_cli():
     bSimple = True
 
     bus.on('speak', handle_speak)
+    connect_to_messagebus()
```

Comment:
bus object now has a `run_in_thread` method that can be used instead of this, if you are cleaning up the code you can basically remove the connect_to_messagebus method now

Reply:
Nice catch, PR updated !